app/views/fact_values/_fact.html.erb

```diff
@@ -8,14 +8,13 @@
         <% end %>
       </td>
   <% end %>
-  <td>
+  <td class="ellipsis">
     <%= fact_name(fact, parent) %>
   </td>
   <td class="fact-col">
     <% unless fact.compose %>
       <%= link_to trunc_with_tooltip(fact.value, 40),
                   fact_values_path("search" => "facts.#{fact.name} = \"#{fact.value}\""),
-                  :title => _("Show all %{name} facts where they are equal to %{value}") % { :name => fact.name, :value => fact.value },
```

Comment:
I think it would be actually better here to leave this tooltip and replace the `trunc_with_tooltip` with `fact.value[0,39]` which explains where this link takes us to. If we want to see the complete value we have the `+` button generated in the next line.